File: api/src/Api/ProductCatalog/Reviews/Requests/ReviewCreateRequest.cs
using System;

namespace DetailingArsenal.Api.ProductCatalog {
    public class ReviewCreateRequest : IDataTransferObject {
        public Guid PadId { get; set; }
        public int Stars { get; set; }
        public int? Cut { get; set; }
        public int? Finish { get; set; }
        public string Title { get; set; } = null!;
        public string Body { get; set; } = null!;
    }
}

File: app/src/api/product-catalog/data-transfer-objects/requests/review-create-request.ts
import { PadCut } from '../pad-cut';
import { PadFinish } from '../pad-finish';
import { Stars } from '../stars';

export interface ReviewCreateRequest {
    padId: string;
    stars: Stars;
    cut: PadCut | null;
    finish: PadFinish | null;
    title: string;
    body: string;
}